lib/fastlane/plugin/android_emulator/actions/android_emulator_action.rb

```diff
@@ -16,33 +16,51 @@ def self.run(params)
         system("#{adb} emu kill > /dev/null 2>&1 &")
         sleep(2)
 
-        UI.message("Creating new emulator")
-        FastlaneCore::CommandExecutor.execute(
-          command: "#{sdk_dir}/tools/bin/avdmanager create avd -n '#{params[:name]}' -f -k '#{params[:package]}' -d 'Nexus 5'",
-          print_all: true,
-          print_command: false
-        )
-
-        UI.message("Override configuration")
-        open("#{Dir.home}/.android/avd/#{params[:name]}.avd/config.ini", 'a') { |f|
-          f << "hw.gpu.mode=auto\n"
-          f << "hw.gpu.enabled=yes\n"
-          f << "skin.dynamic=yes\n"
-          f << "skin.name=1080x1920\n"
-        }
-
-        # Verify HAXM installed on mac
-        if FastlaneCore::Helper.mac?
-          kextstat = Actions.sh("kextstat", log: false)
-
-          UI.user_error! "Please install the HAXM-Extension" unless kextstat.include?("intel")
+        avd_available = false
+        config_file = "#{Dir.home}/.android/avd/#{params[:name]}.avd/config.ini"
+        if File.exists?(config_file)
+          avd_image = File.open(config_file).grep(/^image\.sysdir\.1=/)
+          avd_available = avd_image.any? { |s| s.include?(params[:package].gsub(';', '/')) }
```

Comment:
This should also work:
```ruby
avd_image.any.include? params[:package].gsub(';', '/')
```

Reply:
That didn't work for me -- I got:
```
NoMethodError: [!] undefined method `any' for ["image.sysdir.1=system-images/android-26/google_apis/x86/\n"]:Array
```

Reply:
Yeah, I already edited it. without the any ;-)

Reply:
:-)  That doesn't work either as we're currently doing a partial match on the string. I could change it to the following, but I'm not sure it's that much more readable:
```
avd_available = avd_image.include? "image.sysdir.1=#{params[:package].gsub(';', '/')}\n"
```

Reply:
Ah, i got it working. The problem was we are getting an array

```ruby
avd_image = File.open(config_file).grep(/^image\.sysdir\.1=/).first
avd_available = avd_image.include?(params[:package].gsub(';', '/'))
```

Reply:
that looks better :-)